server/helpers/inputs.js

```diff
@@ -5,7 +5,8 @@
 module.exports = {
   formatter: {
     integer: formatterInteger,
-    date: formatterDate
+    date: formatterDate,
+    nomenclatureFilter: formatterNomenclatureFilter
```

Comment:
`filter` in this contexts may point to wrong conclusions, better just call it `nomenclature`